cmd/minishift/cmd/openshift/config.go

```diff
@@ -27,8 +27,8 @@ const (
 
 var configCmd = &cobra.Command{
 	Use:   "config SUBCOMMAND [flags]",
-	Short: "Displays or patches OpenShift configuration.",
-	Long:  "Displays or patches OpenShift master or node configuration. Patches are supplied in JSON format.",
+	Short: "Displays or updates the OpenShift configuration.",
```

Comment:
First of all, "updating configuration" doesn't always happen with a patch, sometimes you set properties in a config file. This specific usage is with a patch, yes, but the end-goal or overall goal is to update.

We discussed the usage of update/patch before, but if you look at the other config command doc strings you will see that we do use "update" to describe the high-level use-case, and these are already merged and in use on master so I would think that we agreed on this (otherwise we wouldn't have merged those doc strings to begin with).

If you want to change the doc strings across the board we need to discuss further, but it's reverting a convention we already set the last time we reviewed doc strings.